tensorflow_lite_support/python/task/audio/core/tensor_audio_test.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for tensor_audio."""
+import time
+
+import numpy as np
+from absl.testing import parameterized
+from scipy.io import wavfile
```

Comment:
@kinaryml We want to minimize the number of dependencies of the library so let's not use scipy here.

Reply:
@khanhlvg I've just added it here to try out the classifier on the audio recording to preview the audio in the dummy test. I'll remove it later.